### PR TITLE
Changes BaconQRCode for QRCode

### PR DESCRIPTION
**Pull request by PedroCisnerosSantana**
## What does this PR do?

It changes BaconQRCode for QRCode, much more lighter 😉 

## Test Plan

This code might have errors, I need @eldadfux to review and test it ❤️ 

## Related PRs and Issues

#475 

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

 Yes

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -50,7 +50,8 @@
         "domnikl/statsd": "3.0.*",
         "influxdb/influxdb-php": "1.15.*",
         "bacon/bacon-qr-code": "2.0.2",
```
Comment (eldadfux) [inline composer.json:52]: We need to remove this dependency `"bacon/bacon-qr-code": "2.0.2",`.

File: composer.json [hunk 2]
```diff
@@ -50,7 +50,8 @@
...
-        "phpmailer/phpmailer": "6.1.7"
+        "phpmailer/phpmailer": "6.1.7",
+        "chillerlan/php-qrcode": "^4.1"
```
Comment (eldadfux) [inline composer.json:54]: 4.2 seem to be the latest version.

File: app/controllers/api/avatars.php [hunk 3]
```diff
@@ -11,12 +11,15 @@
 use Utopia\Cache\Adapter\Filesystem;
```
Comment (eldadfux) [inline app/controllers/api/avatars.php:11]: Please remove all comments from the code, we can always use other branches or tags to revert.

File: app/controllers/api/avatars.php [hunk 4]
```diff
@@ -366,12 +369,17 @@
...
+       /*  $renderer = new ImageRenderer(
             new RendererStyle($size, $margin),
             new ImagickImageBackEnd('png', 100)
-        );
+        ); */
+        $qropts = new QROptions([
+            'quietzone'         => $size
+        ]);
+        $qrcode = new QRCode($qropts);
+        $qrcode->render($text);
```
Comment (eldadfux) [inline app/controllers/api/avatars.php:380]: This line is not doing anything: `$qrcode->render($text);` just return the result output to no var. What you should do is use the `->send` method in the bottom of the route to attach the image data to our response payload. Example:

```php   
        $response
            ->addHeader('Expires', \date('D, d M Y H:i:s', \time() + (60 * 60 * 24 * 45)).' GMT') // 45 days cache
            ->setContentType('image/png')
            ->send($qrcode->render($text))
        ;
```

**Review by eldadfux — CHANGES_REQUESTED** [review]

File: composer.json (hunk 2 above)
Comment (PedroCisnerosSantana) [reply composer.json:54]: Yes, forgot to apply the change on the VM env, sorry

**Review by eldadfux — APPROVED** [review]

**Pull request merged**